recipes/pfam_scan/build.sh

```diff
@@ -9,8 +9,7 @@ cp -r * $outdir
 # Short wrapper script which sets PERL5LIB while running
 cat > $PREFIX/bin/pfam_scan.pl <<EOF
 #!/bin/bash
-export PERL5LIB=$outdir:$outdir/
-$outdir/pfam_scan.pl \$@
-unset PERL5LIB
+# Last line of script to ensure any failure return code is passed on:
+PERL5LIB=$outdir:$outdir/treg_comparator exec $outdir/pfam_scan.pl "\$@"
```

Comment:
Copy-pasted `:$outdir/treg_comparator` over :wink:. Doesn't really harm, but a single `PERL5LIB=$outdir` should suffice:
```bash
PERL5LIB=$outdir exec $outdir/pfam_scan.pl "\$@"
```
Also, the comments in the wrapper scripts can now be deleted, too. With `exec` the following code shouldn't be executed anyway.

Reply:
Thanks